src/lib/y2storage/yaml_writer.rb

```diff
@@ -440,5 +441,70 @@ def yaml_btrfs_subvolume(subvol)
       content["nocow"] = "true" if subvol.nocow?
       { "subvolume" => content }
     end
+
+    # Return YAML for any unsupported objects in a devicegraph.
+    # Right now this is limited to block devices; this might change in the
```

Comment:
This comment about only block devices is outdated. We also consider `Nfs` objects.

Reply:
Removing that method; it's now used anymore.